src/doc/rustc/src/platform-support.md

```diff
@@ -305,6 +305,7 @@ target | std | host | notes
 `riscv64gc-unknown-fuchsia` |   |   | RISC-V Fuchsia
 `riscv64gc-unknown-linux-musl` |   |   | RISC-V Linux (kernel 4.20, musl 1.2.0)
 [`riscv64gc-unknown-openbsd`](platform-support/openbsd.md) | ✓ | ✓ | OpenBSD/riscv64
+`riscv64-linux-android` |   |   | RISC-V 64-bit Android
```

Comment:
Would you mind filling [this template](https://github.com/rust-lang/rust/blob/master/src/doc/rustc/src/platform-support/TEMPLATE.md) out and dropping a link to it here like the OpenBSD target does above? This very helpful to us for knowing who the maintainers are and some basic steps to troubleshooting issues with the target.

Reply:
I've linked to the existing `platform-support/android.md` file.  My bad for forgetting that in the initial PR.